java/vector/src/test/java/org/apache/arrow/vector/TestBitVectorHelper.java

```diff
@@ -21,12 +21,11 @@
 import static org.junit.Assert.assertTrue;
 import static org.junit.jupiter.api.Assertions.assertFalse;
 
-import org.apache.arrow.memory.ArrowBuf;
 import org.apache.arrow.memory.BufferAllocator;
 import org.apache.arrow.memory.RootAllocator;
 import org.junit.Test;
 
-import io.netty.util.internal.PlatformDependent;
```

Comment:
this should be `siren.io.netty.util.internal.PlatformDependent;`

Reply:
Thanks done